db/keyval/bytes_broker_api.go

```diff
@@ -27,7 +27,7 @@ type BytesBroker interface {
 	// ListKeys is similar to the ListValues the difference is that values are not fetched
 	ListKeys(prefix string) (BytesKeyIterator, error)
```

Comment:
Also need option to specify glob matching in applicable methods, such as ListKeys(), ListValues() and Watch().